Neuro/Recording/DataStructures/Frame.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Neuro.Recording.DataStructures;
+
+[Serializable]
+public record Frame(bool isImposter,
```

Comment:
These are properties, so they should start with an upper case

Reply:
I recommend using the ReSharper add-in to spot naming issues.